rust-doc-style-guide.md

```diff
@@ -0,0 +1,276 @@
+  - Output of previous commands should not start with `$`
+  - For PowerShell-specific examples, use `>` instead of `$`
+
+  [book-ja](https://github.com/rust-lang-ja/book-ja/blob/master-ja/style-guide.md) also suggests:
+
+  - Use `bash` syntax highlighting
+
+- Rust declarations: commands, functions, arguments, parameters, flags, variables
+- In-line command line output
+
+  > Writing a program that prints `Hello, world!`
+
+- Data types: `i8`, `u128`
+- Names of crates, traits, libraries,
+- Command line tools, plugins, packages
+- Files: `config.toml`
+- Architectures and targets
+
+ > The `RISC-V` and `Xtensa` architectures.
+
+### Monospace and Other Types of Formatting
+
+Monospace font can also be used in:
+
+- links
+
+  > [`String`](https://doc.rust-lang.org/std/string/struct.String.html) is a string type provided by ...
+
+- headings
+
+  > ### A `no_std` Rust Environment
+
+
+## Using _Italics_
+
+- Introduce new terms
+
+  > Each value in Rust has an _owner_.
+  > A _hash map_ allows you to associate a value with a particular key.
+
+- Emphasize important concepts or words
+
+  > When `s` comes _into_ scope, it is valid. It remains valid until it goes _out of_ scope.
+
+
+## Mode of Narration
+
+- Use _the first person_ (we) when introducing a tutorial or explaining how things will be done. The reader will feel like being on the same team with the authors working side by side.
+
+  > We'll start writing a program for the LM3S6965, a Cortex-M3 microcontroller. We have chosen this as our initial target because it can be emulated using QEMU so you don't need to fiddle with hardware in this section, and we can focus on the tooling and the development process.
+
+- Use _the second person_ (you) when describing what the reader should do while installing software, following a tutorial or a procedure. However, in most cases you can use imperative mood as if giving orders to the readers. It makes instructions much shorter and clearer.
+
+  > 1\. Install `cargo-generate`
+  >
+  > `cargo install cargo-generate`
+  >
+  > 2\. Generate a new project
+  >
+  > `cargo generate --git https://github.com/rust-embedded/cortex-m-quickstart`
+
+- Use _the third person_ (the user, it) when describing how things work from the perspective of hardware or software
+
+  > A driver has to be initialized with an instance of type that implements a certain `trait` of the embedded-hal which is ensured via trait bound and provides its own type instance with a custom set of methods allowing to interact with the driven device.
+
+
+## Terminology
+
+This chapter lists the terms that have inconsistencies in spelling, usage, etc.
+
+If you spot other issues with terminology, please add the terms here in alphabetical order using the formatting as follows:
+
+- _Recommended term_
+  - Avoid: Add typical phrases in which this term is found
+  - Use: Add recommended phrases
+  - Note: Add more information if needed
+
+### Recommended Terms
```

Comment:
Maybe we should define a term for VS Code (seems like the official wording is Visual Studio Code, but VS Code is also widely used)

Reply:
Added the term